### DISCUSSION TITLE
Notice about XBOX Achievements & DiagTrack

### DISCUSSION THREAD

**Discussion by marvinlehmann** [farag2/Sophia-Script-for-Windows#345] [Ideas]
Sup!

Unfortunately disabling the DiagTrack service (Connected User Experiences and Telemetry) can cause you not being able to get XBOX achievements anymore. It still seems to be a problem although I suffered from this issue years ago.... (╯°□°）╯︵ ┻━┻
I think it would be a good idea to add a small notice/warning about this problem to the _Sophia.ps1_.

**Comment by farag2** [comment]
Totally agree.
